diesel/src/test_helpers.rs

```diff
@@ -1,3 +1,5 @@
+extern crate dotenv;
```

Comment:
This change seems to be unrelated to the other changes.

Reply:
Needed to pull out the PG specific functions to use them for testing.